packages/app/src/components/App.js

```diff
@@ -140,16 +139,7 @@ export class App extends Component {
     }
 
     renderSnackbar() {
```

Comment:
You can get rid of the renderSnackbar function and just include `<Snackbar />` directly in render fn

Reply:
Fixed